go/vt/mysqlctl/proto/schema.go

```diff
@@ -67,10 +63,6 @@ func (sd *SchemaDefinition) String() string {
 	return jscfg.ToJSON(sd)
 }
 
-func (sd *SchemaDefinition) SortByReverseDataLength() {
```

Comment:
This was supposed to be used by data copiers, so they start with the biggest tables and minimize their run time. But you're right that it's now unused. May need to revive it at some point. :)